### PR TITLE
Add Installation for CorpusLoaders

### PR DESCRIPTION
**Pull request by SambhawDrag**
- Added installation instructions for the **CorpusLoaders.jl** package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,17 @@
...
+As per the standard [Julia package installation](https://julialang.github.io/Pkg.jl/dev/managing-packages/#Adding-registered-packages-1):
+
+```julia
+julia> Pkg.add("CorpusLoaders")
+```
+Also, in the Pkg REPL, the package can be added with the add command, as: 
+```julia
+pkg> add CorpusLoaders
+```
+**Note**: Only linux and OS X are supported.
```
Comment (oxinabox) [inline README.md:24]: really? why is that?

**Comment by oxinabox** [comment]
Thanks

File: README.md (hunk 1 above)
Comment (SambhawDrag) [reply README.md:24]: I saw that all of the JuliaText packages had the Installation instructions in their README.md. However this one didn't, so I thought of adding it.

Comment (oxinabox) [reply README.md:24]: Why does this say not compatible with windows?

Comment (SambhawDrag) [reply README.md:24]: Yes, I saw that. 
It was written in a few other packages. 
I'm sorry but I didn't check before the commit. Should I remove the **Note** and commit?

**Comment by SambhawDrag** [comment]
I removed the **Note** and for some reason AppVeyor build failed.

**Comment by SambhawDrag** [comment]
I have removed the note. Please review. :)

**Pull request merged**